src/tests/test_device_manager.cpp

```diff
@@ -174,6 +175,29 @@ TEST_F(DeviceMgrTest, StripedVDevCreation) {
     this->validate_striped_vdevs();
 }
 
+TEST_F(DeviceMgrTest, SmallStripedVDevCreation) {
```

Comment:
Not suggesting it needs to be included in this PR. We can open tickets to add: 
- Mixed drive types with very large drive size (simulated). 
- Pure HDD with very large drive size (32TB). 
- Pure Fast drive with very large drive size (32TB).